test/gol_test.exs

```diff
@@ -1,8 +1,297 @@
 defmodule GolTest do
   use ExUnit.Case
+  import ExUnit.CaptureIO
   doctest Gol
 
-  test "greets the world" do
-    assert Gol.hello() == :world
+  test "tests a dead world is empty" do
+    newWorld = [
+      {{1, 1}, 0},
+      {{1, 2}, 0},
+      {{1, 3}, 0},
+      {{2, 1}, 0},
+      {{2, 2}, 0},
+      {{2, 3}, 0},
+      {{3, 1}, 0},
+      {{3, 2}, 0},
+      {{3, 3}, 0}
+    ]
+
+    assert Gol.worldEmpty?(newWorld) == true
+  end
+
+  test "tests a world with an alive world is not empty" do
+    newWorld = [
+      {{1, 1}, 0},
```

Comment:
data structures, pt 2: now I'm looking at this and I'm thinking -- I don't think this is an optimal data structure for game of life. The majority of our data structure's memory is just there to look at.... 0, nil, nothing.

The only tuple in this list that has any real 'meaning' is `{{2, 1}, 1}`. 

I think from a design perspective we're thinking of grid (maybe a remnant from our past TTT self) when really we could create a stronger design if we thought about it more as the cartesian plane. 

me? I'd probably _only_ store cells that are alive in a map, and refer to them by points/coordinates.